ci-operator/config/openshift/azure-helm/master.json

```diff
@@ -0,0 +1,50 @@
+{
+  "tag_specification": {
```

Comment:
@kargakis if you happen to have a link to where this json spec is defined/documented please can you send it my way?

Reply:
aos-azure way

Reply:
No docs unfortunately atm but they are underway (cc @petr-muller @stevekuznetsov). I can demo what I know in a scrum call or during demo day.

Reply:
possibly https://github.com/openshift/ci-operator/blob/master/pkg/api/types.go is what I was looking for in the shortest term